tasks/buildsources.js

```diff
@@ -16,8 +30,8 @@ module.exports = function(grunt) {
 	var babel = require("babel-core");
 	var mkdir = require('mkdirp').sync;
 
-	grunt.registerTask('buildsources', 'Build polyfill sources', function() {
-
+	grunt.registerTask('buildsources', 'Build polyfill sources', function(done) {
+		var done = this.async();
```

Comment:
Why shadow the `done` parameter here? Maybe just rename the parameter to `_done` instead. I also think this causes deoptimization, but is probably negligible in this case.